vsintegration/src/FSharp.Editor/LanguageService/SymbolHelpers.fs

```diff
@@ -123,7 +122,7 @@ module internal SymbolHelpers =
                             let! sourceText = document.GetTextAsync(cancellationToken) |> Async.AwaitTask
                             let mutable sourceText = sourceText
                             for symbolUse in symbolUses do
-                                match RoslynHelpers.TryFSharpRangeToTextSpan(sourceText, symbolUse.RangeAlternate) with 
+                                match RoslynHelpers.TryFSharpRangeToTextSpan(sourceText, symbolUse) with 
```

Comment:
Is the use of `RangeAlternate` no longer necessary? IIRC it was required for find refs and rename because there were cases where two symbols could represent the same span of text, and `RangeAlteranate` was always the "correct" one.

Reply:
`symbolUse` here is actually a `range` instead of a `FSharpSymbolUse`. The `range` given, named `symbolUse`, is effectively what the `RangeAlternate` value would be.